docs/Shadows.stories.tsx

```diff
@@ -0,0 +1,206 @@
+          Neutral
+        </Text>
+      </ShadowSwatch>
+      <ShadowSwatch
+        style={{
+          boxShadow: 'var(--shadow-size-md) var(--color-primary-shadow)',
+          backgroundColor: 'var(--color-primary-default)',
+          color: 'var(--color-primary-inverse)',
+        }}
+      >
+        <Text as="p" style={{ margin: 0 }}>
+          Primary
+        </Text>
+      </ShadowSwatch>
+      <ShadowSwatch
+        style={{
+          boxShadow: 'var(--shadow-size-md) var(--color-error-shadow)',
+          backgroundColor: 'var(--color-error-default)',
+          color: 'var(--color-error-inverse)',
+        }}
+      >
+        <Text as="p" style={{ margin: 0 }}>
+          Danger
+        </Text>
+      </ShadowSwatch>
+    </div>
+  );
+};
+
+export const Usage = () => {
+  return (
+    <div>
+      <div
+        style={{
+          display: 'grid',
+          gap: '32px',
+          gridTemplateColumns: 'repeat(auto-fill, 200px)',
+          marginBottom: '64px',
+        }}
+      >
+        <ShadowSwatch
+          style={{
+            boxShadow: 'var(--shadow-size-xs) var(--color-shadow-default)',
+          }}
+        >
+          <Text as="p" style={{ margin: 0 }}>
+            Card
+          </Text>
+        </ShadowSwatch>
+        <ShadowSwatch
+          style={{
+            boxShadow: 'var(--shadow-size-sm) var(--color-shadow-default)',
+          }}
+        >
+          <Text as="p" style={{ margin: 0 }}>
+            Dropdown
+          </Text>
+        </ShadowSwatch>
+        <ShadowSwatch
+          style={{
+            boxShadow: 'var(--shadow-size-md) var(--color-shadow-default)',
+          }}
+        >
+          <Text as="p" style={{ margin: 0 }}>
+            Toast
+          </Text>
+        </ShadowSwatch>
+        <ShadowSwatch
+          style={{
+            boxShadow: 'var(--shadow-size-lg) var(--color-shadow-default)',
+          }}
+        >
+          <Text as="p" style={{ margin: 0 }}>
+            Modal
+          </Text>
+        </ShadowSwatch>
+      </div>
+      <div
+        style={{
+          display: 'grid',
+          gap: '32px',
+          gridTemplateColumns: 'repeat(auto-fill, 200px)',
+        }}
+      >
+        <ShadowSwatch
+          style={{
+            boxShadow: 'var(--component-button-primary-shadow)',
+            backgroundColor: 'var(--color-primary-default)',
+            color: 'var(--color-primary-inverse)',
+          }}
+        >
+          <Text as="p" style={{ margin: 0 }}>
+            Button Primary Hover
+          </Text>
+        </ShadowSwatch>
+        <ShadowSwatch
+          style={{
+            boxShadow: 'var(--component-button-danger-shadow)',
+            backgroundColor: 'var(--color-error-default)',
+            color: 'var(--color-error-inverse)',
+          }}
+        >
+          <Text as="p" style={{ margin: 0 }}>
+            Button Danger Hover
+          </Text>
+        </ShadowSwatch>
+      </div>
+    </div>
+  );
+};
```

Comment:
Not sure if we have a standard practice for storybook yet, but i personally think a story with knobs with shadow colors and sizes will be beneficial here

Reply:
I usually save controls for component stories with props but I suppose it could add some value here.  Resolved in https://github.com/MetaMask/design-tokens/pull/137/commits/c55bf5a671de6c4ce3780c7d3ce0ccfac02431d0